src/azul/plugins/metadata/anvil/__init__.py

```diff
@@ -288,3 +295,15 @@ def aggregation_stage(self) -> 'Type[AnvilAggregationStage]':
     @property
     def filter_stage(self) -> 'Type[AnvilFilterStage]':
         return AnvilFilterStage
+
+    def validate_version(self, version: str) -> None:
+        # This metadata plugin uses a `dcp2_datetime_format` version with 'Z' as
```

Comment:
There are no different forms of a DCP/2 version. It [unambiguously](https://github.com/HumanCellAtlas/dcp2/blob/main/docs/dcp2_system_design.rst#312object-naming) only allows one form. That was the entire point. It seems to me that your implementation of `parse_dcp2_datetime` from back then is faulty, and I mentioned that when we discussed this last in PL.

Reply:
I think I'm conflating DCP/2 versions with the `update_date` and `last_modified` properties in metadata entities. Please create two functions called `parse_dcp2_version` and `format_dcp2_version` right next to `parse_dcp2_datetime` and `format_dcp2_datetime` and clarify in the docstrings of these four function what the difference is.

Reply:
Carefully review callsites of the `…_datetime` functions to see if they should be replaced with calls to `…_version`. Use the `…version` ones here for validation.